### PR TITLE
test: add output format for SSB to performance tests

### PR DESCRIPTION
**Pull request by ddelgrosso1**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-storage/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: internal-tooling/performanceTest.ts [hunk 1]
```diff
@@ -37,8 +41,14 @@ export const enum TRANSFER_MANAGER_TEST_TYPES {
   APPLICATION_DOWNLOAD_MULTIPLE_OBJECTS = 'application-download',
 }
 
+const enum OUTPUT_FORMATS {
+  CSV = 'csv',
+  SSB = 'ssb', //Storage shared benchmarking format
```
Comment (danielduhh) [inline internal-tooling/performanceTest.ts:46]: super nit, but IIUC the format is related to Cloud Monitoring, and not necessarily the SSB?

Comment (ddelgrosso1) [reply internal-tooling/performanceTest.ts:46]: I had no idea this was a standardized format. I will adjust naming accordingly.

File: internal-tooling/performanceUtils.ts [hunk 2]
```diff
@@ -224,3 +225,58 @@ export function getValidationType(): 'md5' | 'crc32c' | boolean | undefined {
     return 'md5';
   }
 }
+
+/**
+ * Converts the supplied test results from javascript objects to cloud monitoring format.
+ *
+ * @param {TestResult[]} results array of test iteration results.
```
Comment (shaffeeullah) [inline internal-tooling/performanceUtils.ts:232]: "results array of test iteration results." doesnt make sense to me without me looking through the code again. can you name the function that should be this input?

File: internal-tooling/performanceUtils.ts [hunk 3]
```diff
@@ -224,3 +225,58 @@ export function getValidationType(): 'md5' | 'crc32c' | boolean | undefined {
...
+ *
+ * @returns {AsyncGenerator<string>} a string containing the results of the conversion.
+ */
+export async function* convertToCloudMonitoringFormat(
+  results: TestResult[],
+  bucketName: string
+): AsyncGenerator<string> {
+  for (const curResult of results) {
+    const throughput =
+      curResult.objectSize >= SSB_SIZE_THRESHOLD_BYTES
```
Comment (shaffeeullah) [inline internal-tooling/performanceUtils.ts:243]: can you add a comment explaining the unit conversion happening here?

**Review by shaffeeullah — APPROVED** [review]
two nits, but otherwise looks good. thanks, denis!!

File: internal-tooling/performanceUtils.ts (hunk 3 above)
Comment (ddelgrosso1) [reply internal-tooling/performanceUtils.ts:244]: Added.

**Review by danielduhh — APPROVED** [review]

**Comment by ddelgrosso1** [comment]
Marking do not merge while I do some testing with the SSB utility.

**Pull request merged**